automation/Clean.ps1

```diff
@@ -0,0 +1,51 @@
+
+param(
+    [string]$SwitchName = "Default Switch",
+
+    [Parameter(Mandatory=$true)]
+    [string]$VMName,
+
+    [Parameter()]
+    [string]$Platform = "Hyper-V"
+)
+
+if ($VMName) {
+    try{
+        if ($Platform -eq "Hyper-V") {
+            $dirPath = "C:\Users\$env:USERNAME\.minikube\machines\$VMName"
+            Stop-VM -Name $VMName -Force
+            Remove-Vm -Name $VMName -Force
+        }
+        elseif ($Platform -eq "VirtualBox") {
+            $dirPath = "C:\Users\$env:USERNAME\VirtualBox VMs\$VMName"
+            & VBoxManage controlvm $VMName poweroff --type headless
+            & VBoxManage unregistervm $VMName --delete
+        }
+        Start-Sleep -Seconds 10
+    }
+    catch {
+        Write-Warning "Couldn't stop the $VMName Virtual Machine. It doesn't exist. $_"
```

Comment:
return? Unless we want to continue processing if the above code fails

Reply:
yeah I want to continue processing, since the directory may still exist